Assets/MixedRealityToolkit/Input/Scripts/Cursor/BaseCursor.cs

```diff
@@ -460,9 +463,9 @@ public virtual CursorStateEnum CheckCursorState()
 
                 if (IsHandVisible)
                 {
-                    return TargetedObject != null ? CursorStateEnum.InteractHover : CursorStateEnum.Interact;
+                    return TargetedObject != null ? CursorStateEnum.InteractHover : BaseCursorStateEnum.Interact;
```

Comment:
Where is `BaseCursorStateEnum` defined?

Reply:
That's odd, that should have changed. Will investigate.
Had to rename Cursor to Base Cursor to not impact with the new Cursor namespace.

Reply:
Resolved, bad rename - reverted here. (checked the rest of the code base and this was the only instance)